smartsim/_core/generation/operations/operations.py

```diff
@@ -0,0 +1,276 @@
+import base64
+import os
+import pathlib
+import pickle
+import sys
+import typing as t
+from dataclasses import dataclass, field
+
+from ...commands import Command
+from .utils.helpers import check_src_and_dest_path
+
+# pylint: disable=invalid-name
+entry_point_path = "smartsim._core.entrypoints.file_operations"
```

Comment:
Minor nit: can we do this on a per-name basis instead of disabling the rule for the entire file?
```suggestion
# pylint: disable-next=invalid-name
entry_point_path = "smartsim._core.entrypoints.file_operations"
```

Reply:
oop GAH yes!